src/application.rs

```diff
@@ -163,6 +163,16 @@ impl CoLink {
 
     pub async fn create_entry(&self, key_name: &str, payload: &[u8]) -> Result<String, Error> {
         let mut client = self._grpc_connect(&self.core_addr).await?;
+        #[cfg(feature = "magic_mount")]
+        if key_name.contains("$") {
+            let token = key_name.split("$").last().unwrap();
+            if token.eq("chunk") {
```

Comment:
What's the name of this feature? I saw two names for this feature, `bulk` in the description and function name, but `chunk` here. Could you unify them? Thanks!